torchbenchmark/models/hf_Whisper/__init__.py

```diff
@@ -0,0 +1,19 @@
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+from torchbenchmark.tasks import SPEECH
+import torch
+
+class Model(HuggingFaceModel):
+    task = SPEECH.RECOGNITION
+    # https://cdn.openai.com/papers/whisper.pdf Says for large-v2 they trained on 1024 batch sizes. 
```

Comment:
We should use per-GPU batch size for train here. For example, if the upstream uses 16 GPUs for batch size 1024, then per-GPU batch size is 1024 / 16 = 64.